crates/rpc/Cargo.toml

```diff
@@ -36,6 +36,7 @@ starknet = { workspace = true }
 thiserror = "1.0.38"
 tracing = "0.1.34"
 tracing-subscriber = { version = "0.3.3", features = ["env-filter"] }
+lazy_static = { workspace = true }
```

Comment:
do we still need that?

Reply:
We can move it back from a workspace dependency to a crate specific dependency.